scripts/core/usm.yml

```diff
@@ -47,6 +68,51 @@ etors:
     - name: DEVICE
       desc: "Memory allocation device info"
 --- #--------------------------------------------------------------------------
+type: struct
+desc: "USM allocation descriptor type"
+class: $xUSM
+name: $x_usm_desc_t
+base: $x_base_desc_t
+members:
+    - type: $x_usm_mem_flags_t
+      name:  flags
+      desc: "[in] memory allocation flags"
+    - type: $x_mem_advice_t
+      name: hints
+      desc: "[in] Memory advice hints"
+    - type: "void*"
```

Comment:
I wanted to change this void to `ur_usm_pool_handle_t` but I got an exception when generating the sources: `/home/igor/UR/unified-runtime/scripts/core/usm.yml(76): 'members'[2] 'type' must not be '*_handle_t': $x_usm_pool_handle_t!` Any idea why?

Reply:
@igchor that must be how we have in the scripts. It is just to avoid having in a struct handles, which by the moment we use this struct, might not be defined.

if we want to pass the handle, we can just pass it as another parameter, instead of through the descriptor.

Reply:
@jandres742 why does this apply only to handles, and not for 'flags_t' or 'desc_t' variables?